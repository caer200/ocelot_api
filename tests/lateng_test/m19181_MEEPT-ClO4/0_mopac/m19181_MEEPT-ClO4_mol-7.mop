CHARGES

 
Cl	1.503419 +1 6.398487 +1 15.250072 +1
O	1.007620 +1 5.065469 +1 15.072357 +1
O	0.420016 +1 7.339903 +1 15.092152 +1
O	2.065739 +1 6.535402 +1 16.565388 +1
O	2.517029 +1 6.657484 +1 14.266737 +1
