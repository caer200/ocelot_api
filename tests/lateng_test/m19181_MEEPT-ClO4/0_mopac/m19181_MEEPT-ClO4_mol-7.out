 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 22:31:43 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Cl          1.50341900  *   6.39848700  *  15.25007200  *
     2       O          1.00762000  *   5.06546900  *  15.07235700  *
     3       O          0.42001600  *   7.33990300  *  15.09215200  *
     4       O          2.06573900  *   6.53540200  *  16.56538800  *
     5       O          2.51702900  *   6.65748400  *  14.26673700  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Cl          1.5034      6.3985     15.2501
     2         O          1.0076      5.0655     15.0724
     3         O          0.4200      7.3399     15.0922
     4         O          2.0657      6.5354     16.5654
     5         O          2.5170      6.6575     14.2667

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: O4 Cl  =     5 atoms



      MOLECULAR POINT GROUP   :   C1  

   TOPOGRAPHY OF SYSTEM

   ATOM No.    LABEL   Atoms connected to this atom
      1         Cl      2      3      4      5
      2          O      1
      3          O      1
      4          O      1
      5          O      1

     Ion Atom No.  Type    Charge
 
    1       1     Cl       +1
 
    1       2      O       -1
    2       3      O       -1
    3       4      O       -1
    4       5      O       -1

          COMPUTED CHARGE ON SYSTEM: -3

          COMPUTED CHARGE ON SYSTEM:  -3

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
